dev-docs/bidders/colossusssp.md

```diff
@@ -9,12 +9,21 @@ schain_supported: true
 media_types: banner, video, native
 userIds: britepoolid, identityLink, unifiedId, id5Id
 gdpr: true
+pbjs: true
+pbs: true
 ---
 
-### Bid Params
+### Prebid.JS Bid Params
 
 {: .table .table-bordered .table-striped }
 | Name           | Scope    | Description                                              | Example    | Type      |
 |----------------|----------|----------------------------------------------------------|------------|-----------|
 | `placement_id` | required | Placement Id will be generated on Colossus SSP Platform. | `0`        | `integer` |
 | `traffic`      | optional | Type traffic                                             | `'banner'` | `string`  |
+
+### Prebid.Server Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope    | Description                                              | Example    | Type      |
+|----------------|----------|----------------------------------------------------------|------------|-----------|
+| `TagID` | required | Placement Id will be generated on Colossus SSP Platform. | `'0'`        | `string` |
```

Comment:
This is two different api, and PBS is using newer one which have another naming.

Reply:
Gotcha.

I've noticed that the bidder code used in PBJS (colossusssp) is different than you're using for PBS (colossus). Perhaps it would be best to create a different md file for PBS vs PBJS, with the pbjs and pbs fields set respectively - or another option to change the bidder code to colossusssp in the PBS PR.

Reply:
colossusssp is a little bit to long with to much of 'S'  . Can i create colossu.md in dev/docs for PBS setup, but leave PBS section in colossusssp.md ?

Reply:
> Can i create colossu.md in dev/docs for PBS setup

Yup. colossus.md with a bidder code of colossu for the PBS adapter is good. It would have pbjs: false and pbs:true, with the reverse in this existing colossusssp.md file. 

> but leave PBS section in colossusssp.md ?

I don't recommend leaving a copy fo the PBS section in this file. Perhaps a short note to look at the colossus file for PBS?